website/content/docs/examples/bootstrap-k0s.md

```diff
@@ -71,10 +72,28 @@ bctl apply -f k0s-in-aws-with-tf.yaml
 bctl update -f k0s-in-aws-with-tf.yaml
 ```
 
+5. View the status of the cluster's Kubernetes pods:
+```
+kubectl get pods --all-namespaces
```

Comment:
Lets separate the command from the output so that someone can use the copy button when the page is rendered.

```
<command>
````
You output should look similar to the following:
```
<output>
````